src/Runner.ts

```diff
@@ -2,15 +2,17 @@ import { ProjectFileType } from '@dojo/cli-export-project/interfaces/project.jso
 import Evented from '@dojo/core/Evented';
 import { createHandle } from '@dojo/core/lang';
 import project from './project';
+import * as base64 from './support/base64';
```

Comment:
A related defect is essentially here: https://github.com/dojo/core/issues/299 which I should port over and fix and then refactor here off of core.